src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -389,188 +389,214 @@ public function getJobStatusValues(): array
         ];
     }
 
-    /**
-     * @return FieldList
-     */
-    public function getCMSFields()
+    public function getCMSFields(): FieldList
```

Comment:
We can't update the method signature outside of a major release.

```suggestion
    /**
     * @return FieldList
     */
    public function getCMSFields()
```